bc_obps/service/facility_report_service.py

```diff
@@ -0,0 +1,65 @@
+from uuid import UUID
+from django.db import transaction
+from typing import List, Optional
+
+from registration.models import Activity
+from reporting.models.facility_report import FacilityReport
+from reporting.schema.facility_report import FacilityReportIn, FacilityReportOut
+
+
+class FacilityReportService:
+    @classmethod
+    def get_facility_report_by_version_and_id(
+        cls, report_version_id: int, facility_id: UUID
+    ) -> Optional[FacilityReport]:
+        return FacilityReport.objects.get(report_version_id=report_version_id, facility_id=facility_id)
+
+    @classmethod
+    def get_activity_ids_for_facility(cls, version_id: int, facility_id: UUID) -> List[int]:
+        facility_report = FacilityReport.objects.get(report_version_id=version_id, facility_id=facility_id)
+        return list(facility_report.activities.values_list('id', flat=True))
+
+    @classmethod
+    def get_facility_report_form_data(
```

Comment:
I think this method is not a service method, it's just something to build API response data within the API layer. I'd put it in the `api/facility_report.py` file directly